polygeist/test/polygeist-opt/selecti1ext.mlir

```diff
@@ -0,0 +1,31 @@
+// RUN: polygeist-opt --canonicalize %s | FileCheck %s 
+  
+// CHECK-LABEL: func.func @test1(%arg0: i1) -> i32 {
+// CHECK-DAG:     %c1_i32 = arith.constant 1 : i32
+// CHECK-DAG:     %c2_i32 = arith.constant 2 : i32
+// CHECK-NEXT:    %0 = arith.select %arg0, %c1_i32, %c2_i32 : i32
```

Comment:
Would be sufficient to check that the select operation has not been removed.

Reply:
removed.